spec/requests/people_n_plus_one_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "n+1 queries in the people controller" do
+  describe "#index", n_plus_one: true do
+    context "when viewing people in a collection" do
+      let!(:collection) { create(:collection) }
+
+      populate do |n|
+        create_list(:collection_participant, n, collection: collection)
+      end
+
+      subject do
+        proc do
+          get collection_people_path(collection_id: collection)
+        end
+      end
+
+      warmup { subject.call }
+
+      it "produces a constant number of queries" do
+        expect { subject.call }
+          .to perform_constant_number_of_queries
+      end
+    end
+  end
+
+  describe "#search", n_plus_one: true, pseud_search: true do
+    context "when there are search results" do
+      populate do |n|
+        create_list(:pseud, n, name: "nplusone")
```

Comment:
Based on `works_n_plus_one_spec`, the indexer needs to be prepared for testing:
```suggestion
        PseudIndexer.prepare_for_testing
        create_list(:pseud, n, name: "nplusone")
```

Reply:
That should already be happening because `pseud_search: true` results in https://github.com/otwcode/otwarchive/blob/1d2d228bfa5e5eb8803b38d4b33d56834467d44b/spec/spec_helper.rb#L98